### PR TITLE
Cleanup and make ready for Docker Hub

### PR DESCRIPTION
**Pull request by aheimsbakk**
Optimized Dockerfile and removed unnecessary files for a pycsw base image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,55 +27,60 @@
...
+#
+# Note in the instruction above I added some extra packages:
+# git - to clone pycsw repo or install with pip directly form github
+# bash - for interactive docker session
+# postgresql-dev, psycopg2 - for postgres backend
+# parmap - to easy parallelize routines like mmd2iso
+# xmltodict - pythonic way to work on xml
 
+FROM alpine:3.11
+MAINTAINER Senda <senda@met.no>
```
Comment (mpejcoch) [inline Dockerfile]: s-enda is a project which will cease to exist at some point, is there a better candidate for a maintainer?

Comment (aheimsbakk) [reply Dockerfile]: Good point. I can set me or @epifanio  as maintainer.

Comment (aheimsbakk) [reply Dockerfile]: Fixed updated with adding two maintainers as LABEL, since MAINTAINER is deprecated by Docker.

**Review by epifanio — APPROVED** [review]

**Review by epifanio — APPROVED** [review]
great cleanup!

**Review by epifanio — APPROVED** [review]

**Review by epifanio — APPROVED** [review]

**Review by epifanio — APPROVED** [review]

**Review by epifanio — APPROVED** [review]

**Pull request merged**